modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_docker_container_upload.rb

```diff
@@ -35,34 +35,29 @@ def perform(claim_id) # rubocop:disable Metrics/MethodLength
         # now upload to benefits documents
         start_bd_uploader_job(auto_claim) if auto_claim.status != errored_state_value
       rescue Faraday::ParsingError, Faraday::TimeoutError => e
-        set_errored_state_on_claim(auto_claim)
         set_evss_response(auto_claim, e)
         error_status = get_error_status_code(e)
-        error_message = get_error_message(e)
-
+        error_message = auto_claim.evss_response&.dig(0, 'text')
```

Comment:
@stiehlrod  I'm not sure this is needed anymore, since the error_message is now being set up in the `set_evss_response` method this can probably go, and the line below for the logger that used it can just use `claim.evss_response` in its place, just to make sure they are always exactly the same.

Reply:
Good point @rockwellwindsor-va !